src/templates/hi/2019/base.html

```diff
@@ -0,0 +1,188 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}The Web Almanac वेब रिपोर्ट की एक वार्षिक स्थिति है जो वेब समुदाय की विशेषज्ञता को HTTP संग्रह के डेटा और रुझानों के साथ जोड़ती है।{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac{% endblock %}
+
+{% block skip_navigation %}नेविगेशन स्किप करें{% endblock %}
+
+{% block organization %}Web Almanac HTTP Archive के द्वारा{% endblock %}
+{% block web_almanac_logo %}
+  <span class="wa">Web Almanac</span>
+  <span class="line-group">  
+    <span class="ha">HTTP Archive</span>
+    <span class="pre">के द्वारा</span>
+  </span>
+{% endblock %}
+
+{% block mission %}
+<p>
+हमारा उद्देश्य वेब समुदाय की विशेषज्ञता के साथ HTTP Archive के कच्चे आँकड़ों और रुझानों को जोड़ना है। Web Almanac वेब की स्थिति पर एक व्यापक रिपोर्ट है, जो वास्तविक डेटा और विश्वसनीय वेब विशेषज्ञों द्वारा समर्थित है। यह पृष्ठ सामग्री, यूजर एक्सपीरियंस, प्रकाशन, और वितरण के पहलुओं को फैलाने वाले 20 अध्यायों से युक्त है।
+</p>
+{% endblock %}
+
+{% block read_last_years_almanac %}{{ year | int - 1 }} Web Almanac को पढ़िए{% endblock %}
+
+{% block http_archive_link %}HTTP Archive होम{% endblock %}
+
+{% block page_navigation %}पेज नेविगेशन{% endblock %}
+
+{% block help_translate %}अनुवाद में मदद करें{% endblock %}
+{% block translation_not_available %}अंग्रेजी - अंग्रेजी में उपलब्ध नहीं है{% endblock %}
```

Comment:
According to Google Translate this Translates as "English - not available in English"

It should be "In English - not available in Hindi" and is added after chapters which have not been translated yet.

For example [in Spanish it says](https://almanac.httparchive.org/es/2019/table-of-contents#part-1):

> (en inglés - no disponible en español)

Which means "In English - not available in Spanish"

Reply:
Ok, got it :)